content/en/docs/tasks/administer-cluster/securing-a-cluster.md

```diff
@@ -125,16 +125,16 @@ that the administrator assumed was not in use.
 
 To prevent specific modules from being automatically loaded, you can uninstall them from
 the node, or add rules to block them. On most Linux distributions, you can do that by
-creating a file such as `/etc/modprobe.d/kubernetes-blacklist.conf` with contents like:
+creating a file such as `/etc/modprobe.d/kubernetes-blocklist.conf` with contents like:
```

Comment:
this one is safe to change, it's just a suggested filename.

Reply:
How about `/etc/modprobe.d/kubernetes.conf`?

(`blocklist` could mislead readers or be misread, because modprobe itself uses a different name, but actually you don't need to use either term in the path)